crates/rpc-types/src/genesis.rs

```diff
@@ -1,5 +1,42 @@
 //! OP types for genesis data.
 
+use alloy_serde::OtherFields;
+use serde::de::Error;
+
+/// Info for Optimism chain.
```

Comment:
this should document that this is a container type for all OP specific fields in a genesis file.